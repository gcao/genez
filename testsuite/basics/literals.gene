#!/usr/bin/env gene run

# Test: Various literal types
(print "Integer: " 42)
(print "String: " "Hello")
(print "Boolean true: " true)
(print "Boolean false: " false)
# nil is not yet a keyword, skip for now
# (print "Nil: " nil)