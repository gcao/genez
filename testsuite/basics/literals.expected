Integer:  42
String:  Hello
Boolean true:  true
Boolean false:  false